### ISSUE TITLE
Session verification not carried through to category modification - can't delete/modify them

### ISSUE THREAD

**Issue by yuk75** [SimpleMachines/SMF#2604]
Admin > boards > "modify" category > "delete" / "modify"

"Token verification failed. Please go back and try again."

I know cats are too cute to modify but what if I really wanna? :)

**Comment by XinYenFon** [comment]
I just went to my local board and deleted a board & category.

**Comment by yuk75** [comment]
Seriously? Wonder why it's not working on mine. I downloaded a fresh copy just before posting. It's definitely dropping it from the URL. Did you click through, not just short cut through the menu?

**Comment by XinYenFon** [comment]
Did you opened the page with Right Click > Open in New Tab or Ctrl+Click (you know stuff), that's mostly causing the issue.

**Comment by live627** [comment]
Can't reproduce.

**Comment by yuk75** [comment]
Here's a video, I really don't know how to explain. This is a fresh install. The only thing I have edited is index.php to add this line:
'works' => array('WorkIndex.php', 'WorkIndex'),

And then I added $txt strings to modifications.english.php. Other than that it's just standalone files (four pretty empty files lol: WorkIndex.php and Subs-WorkIndex.php, and WorkIndex.template.php and WorkIndex.English.php)....it was doing it before those files though.

Video: http://www.youtube.com/watch?v=G_23OYb4Spk (helps to include that ha ha)
P.S: The errors are because I put a bunch of strings in and loaded the page without saving the language files so it's my fault not SMFs - so ignore the errors ha ha.

Also I populated my DB with the populate script... I set to 0 attachment and commented that part out cause it was erroring and I made it do like 80 boards and 8 cats cause that's what my board stats said I had lol. http://www.simplemachines.org/community/index.php?topic=422064.msg3275266#msg3275266
